src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/legaladvisor/DocAssemblyTemplateBody.java

```diff
@@ -94,4 +94,10 @@
     private boolean expertReportPermissionPartyGivenToDefendant;
     private List<CCDCollectionElement<String>> expertReportInstructionClaimant;
     private List<CCDCollectionElement<String>> expertReportInstructionDefendant;
+    private String partyName;
+    private CCDAddress partyAddress;
+    private String body;
+    private String caseName;
+    private String caseWorkerName;
```

Comment:
Capitalisation for caseworker is inconsistent with codebase
```suggestion
    private String caseworkerName;
```

Reply:
this is for the time being as to match the new template as this field is caseWorkerName in the template

Reply:
We can update the template though.

Reply:
yes @dharmendrak is working on that..